src/main/java/org/folio/rest/service/ActualCostRecordService.java

```diff
@@ -0,0 +1,58 @@
+package org.folio.rest.service;
+
+import static io.vertx.core.Future.failedFuture;
+import static io.vertx.core.Future.succeededFuture;
+import static java.lang.String.format;
+import static org.folio.rest.jaxrs.model.ActualCostRecord.Status.CANCELLED;
+
+import java.util.Map;
+
+import org.folio.rest.client.CirculationStorageClient;
+import org.folio.rest.exception.FailedValidationException;
+import org.folio.rest.jaxrs.model.ActualCostFeeFineCancel;
+import org.folio.rest.jaxrs.model.ActualCostRecord;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+
+public class ActualCostRecordService {
+  private final CirculationStorageClient circulationStorageClient;
+
+  public ActualCostRecordService(Vertx vertx, Map<String, String> okapiHeaders) {
+    this.circulationStorageClient = new CirculationStorageClient(vertx, okapiHeaders);
+  }
+
+  public Future<ActualCostRecord> cancelActualCostRecord(ActualCostFeeFineCancel entity) {
```

Comment:
Term `entity` is a bit too abstract:
```suggestion
  public Future<ActualCostRecord> cancelActualCostRecord(ActualCostFeeFineCancel cancelationRequest) {
```